updateAzaanTimers.py

```diff
@@ -12,29 +16,48 @@
 system_cron = CronTab(user='pi')
 
 now = datetime.datetime.now()
-strPlayFajrAzaanMP3Command = 'omxplayer -o local /home/pi/adhan/Adhan-fajr.mp3 > /dev/null 2>&1'
-strPlayAzaanMP3Command = 'omxplayer -o local /home/pi/adhan/Adhan-Makkah.mp3 > /dev/null 2>&1'
-strUpdateCommand = 'python /home/pi/adhan/updateAzaanTimers.py >> /home/pi/adhan/adhan.log 2>&1'
-strClearLogsCommand = 'truncate -s 0 /home/pi/adhan/adhan.log 2>&1'
+strPlayFajrAzaanMP3Command = 'omxplayer --vol 1200 -o local {}/Adhan-fajr.mp3 > /dev/null 2>&1'.format(root_dir)
```

Comment:
How do you feel about removing the vol argument from this line and next? People can use their speaker volume instead?

Reply:
You are right. I will introduce them as optional arguments instead.

Reply:
Fixed in ac6ca08.

Reply:
Wait, I will update the README accordingly.

Reply:
Updated in 1a56395.